core/tests/test_extract_parser.py

```diff
@@ -4,7 +4,7 @@
 
 from core.extract import extract_parser
 
-from core.util.audit_logs_parsing import ConnectionLog
+from core.replay.connections_parser import ConnectionLog
```

Comment:
Fix for using connectionLog from `connection_parser.py` and removed `audit_logs_parsing`